server/chat.ts

```diff
@@ -1830,6 +1839,19 @@ export const Chat = new class {
 			handler: commandHandler as AnnotatedChatHandler | null,
 		};
 	}
+	allCommands(table: ChatCommands = Chat.commands, namespace = '') {
+		const results: AnnotatedChatHandler[] = [];
+		for (const cmd in table) {
+			const handler = table[cmd];
+			if (Array.isArray(handler) || !handler || typeof handler === 'string') continue;
+			if (typeof handler === 'object') {
+				results.push(...this.allCommands(handler, `${namespace}${cmd} `));
+				continue;
+			}
+			results.push(handler as AnnotatedChatHandler);
+		}
+		return results;
+	}
```

Comment:
```suggestion
	allCommands(table: ChatCommands = Chat.commands) {
		const results: AnnotatedChatHandler[] = [];
		for (const cmd in table) {
			const handler = table[cmd];
			if (Array.isArray(handler) || !handler || typeof handler === 'string') continue;
			if (typeof handler === 'object') {
				results.push(...this.allCommands(handler));
				continue;
			}
			results.push(handler as AnnotatedChatHandler);
		}
		return results;
	}
```

`namespace` appears unused.